CHANGELOG.md

```diff
@@ -3,6 +3,344 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.0.0-alpha.9
+
+_Jan 18, 2024_
+
+We'd like to offer a big thanks to the 11 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 The Data Grid headers are now sticky - which brings immense improvements to scrolling, state management, and overall performance of the grid
```

Comment:
I think we can abstract the solution here, specially considering we may have a gif to showcase.

```suggestion
- 🎁 The Data Grid headers have been refactored to bring immense improvements to scrolling, state management, and overall performance of the grid.
```